templates/CWP/CWP/PageTypes/Layout/NewsHolder.ss

```diff
@@ -1,27 +1,34 @@
 <div class="container">
     <div class="row">
-        <div class="col-lg-12 mb-4">
+        <div class="col-lg-12">
             <div class="page-header">
                 $Breadcrumbs
                 <h1>$Title</h1>
             </div>
-            <% if $Content.RichLinks %>
-                $Content.RichLinks
-            <% else %>
-                $Content
-            <% end_if %>
-            <% include NewsFilterContext %>
         </div>
 
-        <section class="listing <% if $AvailableMonths || not $FilteredUpdates %>col-lg-8<% else %>col-lg-12<% end_if %>">
-            <% if $FilteredUpdates %>
-                <% include FilteredUpdates ControllerName=$ClassName %>
-            <% else %>
-                <article>
-                    <p><%t CWP_FilteredUpdates.NoNews "No news" %></p>
-                </article>
-            <% end_if %>
-        </section>
+        <div class="col-8">
+            <div class="row">
```

Comment:
Instead of `row` and `col-lg-*` I think you should be able to get the same results using just section tags.

ie.

```
<section class="mb-3">
</section>
<section class="listing">
</section>
```

Reply:
Done!